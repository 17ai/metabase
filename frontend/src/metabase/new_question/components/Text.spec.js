import React from "react";
import renderer from "react-test-renderer";

import Text from "./Text";

describe("Text", () => {
    it("should render correctly", () => {
        const tree = renderer
            .create(<Text>Why hey here's some body text</Text>)
            .toJSON();

        expect(tree).toMatchSnapshot();
    });
});
