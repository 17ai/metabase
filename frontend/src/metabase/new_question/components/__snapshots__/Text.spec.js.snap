exports[`Text should render correctly 1`] = `
<p>
  Why hey here\'s some body text
</p>
`;
